app/scripts/shell/dimAppCtrl.controller.js

```diff
@@ -198,6 +198,13 @@
 
     // Don't refresh more than once a minute
     var refresh = _.throttle(vm.refresh, 60 * 1000);
+    
+    vm.downloadCsvFiles = function(){
+      //TODO better method for grabbing this?
```

Comment:
could consider doing something like:

```
vm.downloadCsvFiles = function() {
    dimCsvService.downloadCsvFiles(dimStoreService.getStores());
};
```

or since we're including the `dimStoreService` in the `dimCsvService` we could just load it in there. either way is fine though.